Content.Shared/Ninja/Components/StunProviderComponent.cs

```diff
@@ -49,13 +49,13 @@ public sealed partial class StunProviderComponent : Component
     /// Time that someone is stunned for, stacks if done multiple times.
     /// </summary>
     [DataField]
-    public TimeSpan StunTime = TimeSpan.FromSeconds(5);
+    public TimeSpan StunTime = TimeSpan.FromSeconds(1.5); // Harmony: Reduced to fit flash stun time
 
     /// <summary>
     /// How long stunning is disabled after stunning something.
     /// </summary>
     [DataField]
-    public TimeSpan Cooldown = TimeSpan.FromSeconds(2);
+    public TimeSpan Cooldown = TimeSpan.FromSeconds(2); // Harmony: Stun cooldown changed to prevent stunlocking
```

Comment:
This value is now unchanged from upstream.
Either a mistake, or comment should be removed?